package-lock.json

```diff
@@ -6455,14 +6437,6 @@
       "integrity": "sha1-P7z7FbiZpEEjs0ttzBi3JDNqLK4=",
       "dev": true
     },
-    "pad": {
```

Comment:
Latest release of `csv-parse` drops the dependency on `pad` (actually drops /all/ dependencies)
* https://github.com/adaltas/node-csv-parse/blob/v4.6.3/package.json#L87 vs
* https://github.com/adaltas/node-csv-parse/blob/v4.10.1/package.json#L85

The `pad` dependency had `wcwidth` -> `defaults` -> `clone` as transients.